vowpalwabbit/core/src/reductions/search/search_multiclasstask.cc

```diff
@@ -24,7 +24,10 @@ void initialize(Search::search& sch, size_t& num_actions, VW::config::options_i&
 {
   task_data* my_task_data = new task_data();
   sch.set_options(0);
-  sch.set_num_learners(num_actions);
+  size_t num_learner_ceil = 1;
+  while (num_learner_ceil < num_actions) { num_learner_ceil *= 2; }
+  // TODO: temporary fix to ensure enough learners, look into case where learner is created outside bounds
+  sch.set_num_learners(num_learner_ceil);
```

Comment:
This makes me nervous...

Reply:
emailed john about this with 5 or so options, agreed this is the most suitable solution